src/app/add-item/add-item.page.ts

```diff
@@ -11,21 +12,26 @@ import { Observable } from 'rxjs';
 export class AddItemPage implements OnInit {
   categories$: Observable<Category[]>;
   categories: Category[];
+
+  categoryName;
```

Comment:
I used this variable instead of `budget.category.name` because since we maintain categories separately, I felt it made more sense to grab the latest copy of a the specified category, instead of defaulting `budgetItem.category` to an empty `Category`

Reply:
Makes sense to me